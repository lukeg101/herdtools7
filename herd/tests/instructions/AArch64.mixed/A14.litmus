AArch64 A14
(* Store register, with offset, symbolic location *)

 { uint64_t x; 0:x0=1; 0:x2=x; int64_t 0:x3=4;}

P0;
  STR W0, [X2, X3];

exists (x=4294967296)
