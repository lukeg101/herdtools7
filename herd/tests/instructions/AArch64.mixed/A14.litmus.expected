Test A14 Allowed
States 1
x=0x100000000;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=0x100000000)
Observation A14 Always 1 0
Hash=9089f4ae66e1a6cd2953c77e864cd058

